rules/do-not-provide-archors-for-feedback/rule.md

```diff
@@ -0,0 +1,34 @@
+---
+type: rule
+title: Do you know to not give anchors for getting feedback?
+uri: do-not-provide-archors-for-feedback
+authors:
+  - title: Adam Cogan
+    url: https://www.ssw.com.au/people/adam-cogan
+created: 2023-03-22T15:25:02.074Z
+guid: 6d9db0ae-8728-42d3-89e8-7f7369e5faf7
+---
+Anchors are prompts or questions that guide the person providing feedback towards specific areas. 
+
+When it comes to receiving feedback, it's natural to want to guide the conversation by providing anchors that direct the conversation towards specific topics/elements. While this approach may seem helpful, it can actually hinder the quality of feedback received.
+
+<!--endintro-->
+
+For example, if you were testing a new website, you might give an anchor such as *"What do you think of the color scheme?"* to direct the feedback towards the visual design. This approach can limit the feedback you receive by steering the conversation away from potentially valuable insights.
+
+One of the most effective ways to get high-quality feedback is to **observe how people use/do something**. 
+
+For example, if you're testing a new product, you might watch someone use it and take notes on what they struggle with or what they seem to enjoy. This approach allows you to get feedback that is unbiased and focused on the user experience.
+
+`youtube: https://www.youtube.com/embed/fjYtzGLnH8k`
+**Video: How to Test a Frying Pan at the Store? 🤣 (15 s)**
+
+Another effective approach is to [ask open-ended questions](/ask-open-ended-questions/) that allow the person providing feedback to offer their thoughts freely.
+
+By taking this approach, you can receive more valuable and constructive feedback that will ultimately lead to better outcomes.
+
+### Written work
+
+When it comes to receiving feedback on emails, it's best to get the person providing feedback to read it themselves, rather than reading it to them. 
```

Comment:
I think "When it comes to receiving feedback" should read "When it comes to requesting feedback"